components/scream/tests/scream_cldfraction/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+INCLUDE (ScreamUtils)
```

Comment:
Does a cld_fraction standalone test makes sense? What are we testing here? If the impl is all we care, we could just put tests in `physics/cld_fraction/tests`, without bothering the with AD. Seems like a lot of work to setup a test.

Reply:
But that work is already done, right? And the more tests the better? I think this test just confirms that the code doesn't crash... Seems like a fine start.

Other tests could be: can we ever get cld_fraction>1 or <0? If qi>>1e-5 everywhere, is ice cldfrac=1 everywhere?. if qi=0 everywhere, is ice cldfrac=0 everywhere? If alst>aist, is ast=aist in cldfraction_calc_totalfrac? and vice versa.

Reply:
Yes, the last step would be to put in unit tests.  I haven't done this yet.

Reply:
Well, Aaron already put in the effort, so I guess it's ok to keep them. And I guess it does test something more than just an individual function test, namely that hooks to the AD are correctly set up. Alright, I did not think that through.